### ISSUE TITLE
Probleme installing the extension

### ISSUE THREAD

**Issue by RonoBlag** [EverMCServer/vscode-BetonQuest#6]
Hello, I'm trying to install your extensions, but after installing everything, this buton isn't showing up on my visualstudio Code. 
![screenshot-conversation-light](https://github.com/EverMCServer/vscode-BetonQuest/assets/98655548/c6313d3a-23bd-427c-a18e-7994b59d9d24)
Do you have maybe an idea of why, it would be of great help for me.

Thank You

**Comment by k-jiang** [comment]
Are you working with [the 2.0-DEV scripts format](https://docs.betonquest.org/2.0-DEV/Documentation/Scripting/Packages-%26-Templates/)? This extension is not compatible with it yet. [I'm still working on it](https://github.com/EverMCServer/vscode-BetonQuest/issues/5) but don't expect it will come out soon.

For now if you want to use this extension with your existing 2.0-DEV files, please try these work-around steps:

1. Copy only one of your Conversation content onto a speared `.yml` file:
```yaml
conversations:
  Jones: # script's name
    quester: "Jones"
    NPC_options:
      noWoodInInv:
        text: "Looks like you don't have the required wood with you. Bring me 20 oak
            logs!"
        conditions: "startedTag,woodcuttingDoneTag,!logsInInventory"
      # ... more options ...
```
2. Delete the leading `conversations` and script's name completely. Don't forget to un-indent the whole script so it will not report any errors
```yaml
# Select all and press `Shift + Tab` twice to un-indent, or until `NPC_options` is aligned to the most left side
quester: "Jones"
NPC_options:
  noWoodInInv:
    text: "Looks like you don't have the required wood with you. Bring me 20 oak
        logs!"
    conditions: "startedTag,woodcuttingDoneTag,!logsInInventory"
  # ... more options ...
```
3. If your Conversation does not have a `first` section, you should create one and fill in the name of your first option
```yaml
quester: "Jones"
first: "noWoodInInv" # The initial starting option of your conversation
NPC_options:
  noWoodInInv:
    text: "Looks like you don't have the required wood with you. Bring me 20 oak
        logs!"
    conditions: "startedTag,woodcuttingDoneTag,!logsInInventory"
  # ... more options ...
```
4. With the extension installed, goto the separated `.yml` file then press `Ctrl + Shift + P` (`Command + Shift + P` for MacOS)
5. Enter `Open Conversation Flowchart` then hit `Enter`

Now you should have the Flowchart opened on the side view.

**Comment by k-jiang** [comment]
I have just published 0.1.0. It is now support the 2.0 format. Please try it out.
